Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -1657,18 +1675,13 @@ def resource_list_command(client: Client, args: Dict[str, Any]) -> CommandResult
     for response_item in response_items:
         change_timestamp_to_datestring_in_dict(response_item)
 
-    readable_responses = deepcopy(response_items)
-    nested_headers = {'resourceListType': 'Type'}
-    for readable_response in readable_responses:
-        extract_nested_values(readable_response, nested_headers)
-
-    headers = ['name', 'id', 'Type', 'description', 'lastModifiedBy']
+    headers = ['name', 'id', 'resourceListType', 'description', 'lastModifiedBy']
     command_results = CommandResults(
         outputs_prefix='PrismaCloud.ResourceList',
         outputs_key_field='id',
-        readable_output=f'Showing {len(readable_responses)} of {total_response_amount} results:\n'
+        readable_output=f'Showing {len(response_items)} of {total_response_amount} results:\n'
                         + tableToMarkdown('Resources Details:',
```

Comment:
Do we want to add the automatic nested handling?

Reply:
It was here only in order to change the title of `resourceListType` to `Type`. I reverted it because this is the title in the UI.